### PR TITLE
Adding acceptance test for GOV notify fulfilment request

### PR DESCRIPTION
**Pull request by hbrace**
# Motivation and Context

Adding tests for the new notify-processor-service to test UAC fulfilment requests.

# What has changed

Scenario added to check notify api gets called.

# How to test?

Run these tests with census-rm-notify-processor and census-rm-notify-stub on same branch names.
https://github.com/ONSdigital/census-rm-notify-processor/pull/1
# Links

https://trello.com/c/yX4y7G8z/944-rmc-145b-implement-uac-fulfilment-event-part-a-13-part-b-5#

# Screenshots (if appropriate):

### CODE REVIEW

**Comment by hbrace** [comment]
[RMC-145(B.1) Implement UAC fulfilment event (13)](https://trello.com/c/yX4y7G8z/944-rmc-145b-implement-uac-fulfilment-event-part-a-13-part-b-5)

**Comment by hbrace** [comment]
[RMC-145(B.3) Implement UAC fulfilment event - Acceptance tests? (5)](https://trello.com/c/RSo0ELek/1033-rmc-145b3-implement-uac-fulfilment-event-acceptance-tests-5)

File: acceptance_tests/features/fulfilment_request.feature [hunk 1]
```diff
@@ -4,4 +4,12 @@ Feature: Handle fulfilment request events
     Given sample file "sample_input_england_census_spec.csv" is loaded
     And messages are emitted to RH and Action Scheduler with [01] qids
     When a fulfilment request message for a created case is sent
-    Then a fulfilment request event is logged
\ No newline at end of file
+    Then a fulfilment request event is logged
+
+
+  Scenario: Log event when a fulfilment request event is received
```
Comment (NickGrantX) [inline acceptance_tests/features/fulfilment_request.feature]: This scenario name needs to change (it's a dupe) to refer to the UAC SMS.

**Review by NickGrantX — APPROVED** [review]
I paired on this but it looks good and works for me 👍

**Review by AdamHawtin — COMMENTED** [review]
LGTM but should we be testing all the fulfilment codes (making sure the questionnaire ID is correct for each etc.)?

**Review by AdamHawtin — APPROVED** [review]
👍 Established all the UAC fulfilment codes are tested at a lower level

**Pull request merged**